### PR TITLE
episode 13 - Setting up HashiCorp Vault

### PR DESCRIPTION
**Pull request by Andrey9kin**

### CODE REVIEW

File: content/episodes/Setting up HashiConf Vault.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Setting up HashiConf Vault"
```
Comment (veggiemonk) [inline content/episodes/Setting up HashiConf Vault.md]: What about "setting up and hardening" ?

**Review by veggiemonk — APPROVED** [review]
Nice just one question about the title

File: content/episodes/Setting up HashiConf Vault.md (hunk 1 above)
Comment (Andrey9kin) [reply content/episodes/Setting up HashiConf Vault.md]: @veggiemonk maybe even Getting started with HashiСorp Vault. What do you think?

Comment (mattiashem) [reply content/episodes/Setting up HashiConf Vault.md]: What you need to know abount HashiCorp Vault

Comment (veggiemonk) [reply content/episodes/Setting up HashiConf Vault.md]: Up to you, I have no strong opinion about that

**Pull request merged**